### PR TITLE
Tiny Barbarian DX

### PR DESCRIPTION
**Pull request by Dantttt**
Updated json with no errors is in new commit named Create Tiny Barbarian DX
boxart

### CODE REVIEW

File: info.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+ { 
+{
```
Comment (firewolfgames) [inline info.json]: Why this extra bracket?

**Comment by Dantttt** [comment]
this is old commit look at the new of today it's fixed all

**Comment by AcK77** [comment]
I don’t what the file named "Tiny Barbarian DX" is? But at least your state is the current one so we don’t need to update it!

**Comment by Dantttt** [comment]
Have you updated?

**Pull request closed without merging**